exdb/models.py

```diff
@@ -239,3 +278,35 @@ def send(self, *args, **kwargs):
 
     def __str__(self):
         return self.name
+
+
+class Semester(models.Model):
+    start_datetime = models.DateTimeField()
+    end_datetime = models.DateTimeField()
+
+    @classmethod
+    def get_current(cls):
+        current_time = now()
+        semester = cls.objects.filter(start_datetime__lte=current_time, end_datetime__gte=current_time).first()
+        if semester is None:
+            semester = cls.objects.filter(start_datetime__lte=current_time).order_by('-end_datetime').first()
+        if semester is None:
+            semester = cls.objects.order_by('-start_datetime').first()
+        return semester
+
+    def __str__(self):
+        return str(self.start_datetime.strftime("%B %d, %Y") + " - " + self.end_datetime.strftime("%B %d, %Y"))
+
+
+class Requirement(models.Model):
+    start_datetime = models.DateTimeField()
```

Comment:
If we change this to a `DateField`, it would create an ambiguous comparison between Experience's start/end datetimes and Requirement's start/end datetimes.